gammapy/cube/core.py

```diff
@@ -369,6 +376,23 @@ def sky_image(self, energy, interpolation=None):
             data = self.data[int(z)]
         return SkyImage(name=self.name, data=data, wcs=self.wcs)
 
+    def sky_image_idx(self, idx):
+        """
+        Slice a 2-dim `~gammapy.image.SkyImage` from the cube at a given index.
+
+        Parameters
+        ----------
+        idx : int
+            Index of the sky image.
+
+        Returns
+        -------
+        image : `~gammapy.image.SkyImage`
+            2-dim sky image
+        """
+        data = self.data[idx]
+        return SkyImage(name=self.name, data=data, wcs=self.wcs)
```

Comment:
Return copy for data and wcs, at least by default, to avoid accidentally coupled objects?
(I thought we had that method already, didn't review the SkyCube class again. Just ignore me if this is not a good suggestion)